__tests__/app.test.js

```diff
@@ -348,3 +347,80 @@ describe.only("GET: /api/users", () => {
       });
   });
 });
+
+describe("GET: api/reviews (queries)", () => {
+  it("should return all reviews with matching queried category", () => {
+    return request(app)
+      .get("/api/reviews/?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toEqual([
+          {
+            title: "Jenga",
+            designer: "Leslie Scott",
+            owner: "philippaclaire9",
+            review_id: 2,
+            review_img_url:
+              "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+            review_body: "Fiddly fun for all the family",
+            category: "dexterity",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+          },
+        ]);
+      });
+  });
+  it("should sort correctly by input sort_by value", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=created_at")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted("created_at");
+      });
+  });
+  it("should sort correctly by input order value", () => {
+    return request(app)
+      .get("/api/reviews?order=ASC")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted(`ASC`);
+      });
+  });
+  it("should default to sort by created_at", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted("created_at");
+      });
+  });
+  it("should default to desc", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted("DESC");
+      });
+  });
+  it("400: rejects trying to sort by an invalid value", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=DROP")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(`DROP is not a valid sort_by value.`);
+      });
+  });
+  it("400: rejects trying to order by an invalid value", () => {
+    return request(app)
+      .get("/api/reviews?order=MONKEY")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(`MONKEY is not a valid order value.`);
+      });
```

Comment:
Nice one James, just a couple of extra considerations to think about:
- 404 for category not found
- 200 for category valid and exists but there happens to be no reviews associated with that category just yet. Not an error so we need to separate that from any other logic related to the response length